src/SearchResults.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import moment from "moment";
+import CustomerProfile from "./CustomerProfile";
+
+const SearchResult = props => {
+  const [selectedId, setSelectedId] = useState();
+
+  const setSelected = id => {
```

Comment:
I don't think you need this function, as you can just pass the state hook to your `TableRow`, i.e.
```
<TableRow item={item} index={index} setSelected2={setSelectedId} />
```
Also, `setSelected2` is not a very clear name for a function

Reply:
Thank you Ellie, it makes sense, I changed it .